projects/aca-folder-rules/src/lib/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -121,6 +133,11 @@ export class RuleSimpleConditionUiComponent implements ControlValueAccessor, OnD
     if (!this.selectedFieldComparators.find((comparator) => comparator.name === this.comparatorControl.value)) {
       this.comparatorControl.setValue('equals');
     }
+    if (!this.valueControl.value && this.selectedField?.name === 'mimetype') {
```

Comment:
missing unit tests for everything new you added

Reply:
test cases in progress

Reply:
unit tests added